.github/workflows/publish-abac-nuget.yml

```diff
@@ -0,0 +1,36 @@
+name: Pack and publish Altinn.Authorization.ABAC nuget package
+
+on:
+  pull_request:
+    branches: infrastructure/publish-nuget-action
+  release:
+    types:
+      - published
+
+jobs:
+  build-pack:
+    #if: startsWith(github.ref, 'refs/tags/Altinn.Authorization.ABAC-')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: |
+            2.0.x
+      - name: Install deps
+        run: |
+          cd src/Altinn.Authorization.ABAC
```

Comment:
Instead of using cd in the commands, set the working directory.

See: https://github.com/Altinn/altinn-authorization-utils/blob/a1deeb53af9573579aee66e0f6710691f2ac4c3f/.github/workflows/release.yml#L80